test/gtest/ucp/test_ucp_memheap.cc

```diff
@@ -7,6 +8,134 @@
 #include "test_ucp_memheap.h"
 
 
+void test_ucp_memheap::test_nonblocking_implicit_stream_xfer(nonblocking_send_func_t send, size_t alignment)
```

Comment:
please don't copy/paste the code from test_blocking xfer, but extract common code to a function